datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/service_checks.py

```diff
@@ -64,6 +64,21 @@ def service_checks():
                     file_failed = True
                     display_queue.append((echo_failure, '  Attribute `{}` is required'.format(attr)))
 
+                # integration name matches display_name from manifest.json
+                manifest_json = load_manifest(check_name)
+                if not manifest_json:
+                    file_failed = True
+                    display_queue.append((echo_failure('{}/manifest.json doesn\'t exist'.format(check_name))))
+                    echo_info('{}/manifest.json... '.format(check_name), nl=False)
+                elif manifest_json.get('display_name').lower() != service_check.get('integration').lower():
```

Comment:
```suggestion
                elif manifest_json.get('display_name') != service_check.get('integration'):
```

Reply:
The casing doesn't appear to matter to the internal tooling. There are a significantly larger number of integrations that break if we don't do `lower`. And I don't think its an issue if they have different casing.

Reply:
I say let's not depend on backend internals and not normalize please